src/settings.cpp

```diff
@@ -2013,14 +2018,13 @@ static uint GetCompanySettingIndex(const SettingDesc *sd)
  */
 bool SetSettingValue(const SettingDesc *sd, int32 value, bool force_newgame)
 {
-	if ((sd->flags & SGF_PER_COMPANY) != 0) {
+	const IntSettingDesc *sdb = sd->IntSetting();
```

Comment:
hihi, @ `sdb` .. `SettingDescBase` is back baby :)

I have no suggestions for a better name :P